src/__experimental__/components/checkbox/__snapshots__/checkbox-group.spec.js.snap

```diff
@@ -250,14 +254,16 @@ exports[`CheckboxGroup renders as expected 1`] = `
       </div>
     </div>
     <div
+      checked={false}
       className="c5 c6"
       data-component="checkbox"
-      name="check-optional"
+      name="my-checkbox-group"
```

Comment:
We shouldn't be setting the `name` attribute of a `<div>`. Name should only be an attribute of a form input element 
https://www.w3schools.com/tags/att_name.asp

Reply:
resolved in 007bc48b109edd5a4bea662ebcd953f4253334b3